<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Charts</string>
     </attribute>
     <widget class="QTextBrowser" name="chartBrowser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>611</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="chartWidgetGroupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>681</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QPushButton" name="generateChartButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelChart">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Chart:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="outputComboBox">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>10</y>
         <width>131</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="chartBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>391</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Tab">
     <attribute name="title">
      <string>Tab</string>
     </attribute>
     <widget class="QTextBrowser" name="tabBrowser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>671</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="tabWidgetGroupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>681</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QGroupBox" name="sourceMusicSelectionGroup">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>641</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Source Music</string>
       </property>
       <widget class="QRadioButton" name="sourceExpressionButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Expression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="expressionEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>531</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Musical Expression (see help)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="sourceFileButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
       <widget class="QLabel" name="sourceFileLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>471</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QToolButton" name="fileExplorerButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>60</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Downloads/folder_icon.png</normaloff>../../../../Downloads/folder_icon.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="transposeGroup">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>641</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Transpose</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Half Steps</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="transposeSpinner">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-96</number>
        </property>
        <property name="maximum">
         <number>96</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="sourceKeyCheckBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>From Key</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sourceKeyBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>91</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="transposeDirectionRadioButtons">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="title">
         <string>Direction:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QRadioButton" name="transposeClosestButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>0</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Closest</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="transposeUpButton">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>0</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="transposeDownButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>0</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="goButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>200</y>
         <width>101</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="harpGroupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QComboBox" name="typeBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>341</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Harp</string>
     </property>
    </widget>
    <widget class="QComboBox" name="harpKeyBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionSaveOutput">
   <property name="text">
    <string>&amp;Save Output</string>
   </property>
   <property name="toolTip">
    <string>Save generated output to file</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Meta+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
